docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM registry.access.redhat.com/ubi8/python-39:latest
+
+# We need to be root to setup the container image
+USER 0
+
+# Install required packages and setup ssh
+RUN yum install -y yum-utils openssh-server git sudo curl && \
+	sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
+	ssh-keygen -t rsa -f /etc/ssh/ssh_host_rsa_key -N '' && \
+	ssh-keygen -t dsa -f /etc/ssh/ssh_host_dsa_key -N ''
+
+# Copying default config files and inmanta user login shell
+COPY resources/shell-with-env /usr/bin/shell-with-env
+COPY resources/server.cfg /etc/inmanta/inmanta.cfg
```

Comment:
Because we copy this before we do the install, we generate an .rpmnew file.

```
(app-root) bash-4.4$ ls -l /etc/inmanta/
total 8
-rw-r--r--. 1 1000 1000 3001 Nov 26 08:57 inmanta.cfg
-rw-r--r--. 1 root root 3402 Feb 10 13:58 inmanta.cfg.rpmnew
```

(This was so for the iso container)

Reply:
Done